earth_enterprise/src/third_party/gdal/SConscript

```diff
@@ -26,26 +26,11 @@
 # lines below.
 
 Import('third_party_env')
-import os
```

Comment:
These mrSID changes are now in master. Eventually you want to get them by updating your fork  or just ensuring that we get them when this PR is merged.